common/dropwizard/src/main/java/com/bazaarvoice/emodb/common/dropwizard/guice/SystemTablePlacement.java

```diff
@@ -1,4 +1,4 @@
-package com.bazaarvoice.emodb.table.db.astyanax;
+package com.bazaarvoice.emodb.common.dropwizard.guice;
```

Comment:
Why the move?  Was "common/astyanax" not a dependency for everywhere this was needed?  Or was it just to separate the concept of system table placement from astyanax?  Not objecting to the move, just curious for the reasoning behind it.  Also not a huge fan of putting it in the "guice" package, seems a little generic, but can't think of anywhere better to put it and I see there's already some global bindings there.

Reply:
Both actually. The emodb-job module did not have access to it, as it was previously located in the table/db/astyanax, not common/astyanax. It seemed like a better idea to relocate SystemTablePlacement then to rope in another dependency that wasn't really necessary. When relocating it, I felt like it actually had nothing to do with astyanax, and the other global guice annotations were in the guice package, so I figured that would be the best place to put it.